test/AudioTest.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import chai, {expect} from 'chai';
 import {shallow, mount} from 'enzyme';
-import Audio from '../src/components/Audio';
-import Transformation from '../src/components/Transformation';
+import cloudinary from './cloudinary-proxy';
```

Comment:
Out of curiousity, what's cloudinary-proxy?

Reply:
It abstracts and exposes one of lib/dist/src as the test subject. This is what i'd like to simplify in the future

Reply:
* simplify while removing lib

Reply:
Okay,

I guess in the future we could also import from src/index, that would probably be the same thing, but testing on the dist adds another layer of assurance.